Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1156,17 +1156,34 @@ def appendContext(key, data, dedup=False):
     if data is None:
         return
     existing = demisto.get(demisto.context(), key)
+
     if existing:
-        strBased = isinstance(data, STRING_TYPES) and isinstance(existing, STRING_TYPES)
-        if strBased:
-            data = data.split(',')
-            existing = existing.split(',')
-        newVal = data + existing
+        if isinstance(existing, STRING_TYPES):
+            if isinstance(data, STRING_TYPES):
+                new_val = data + ',' + existing
+            else:
+                return_error("Cannot append data to the existing context - \n The data is of type {} while the "
```

Comment:
Don't use return_error in helper functions. Instead you can just try to add strings, it will raise `TypeError: can only concatenate list (not "str") to list` which is self explanatory

Reply:
Done